api/src/multiaccessy/accessy.py

```diff
@@ -416,13 +432,23 @@ def _get_org_user_from_phone(
             return None
 
 
-accessy_session = AccessySession()
+accessy_session = AccessySession() if AccessySession.is_env_configured() else None
+
 
+(
+    STATUS_OK,
+    ERROR_NOT_CONFIGURED,
+) = range(0,2)
```

Comment:
This is literally twice as many lines as defining them in the boring and normal way :p

Reply:
ExTeNsIbiliTyyy hahaha yeah its over the top because I initially thought of placing them at the top, suggesting to be use in the rest of the file, but we have better objects than integers to represent errors in python ^^